### DISCUSSION TITLE
Mini-Cart Update Hooks

### DISCUSSION THREAD

**Discussion by stickypages** [elementor/elementor#22386] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

We need some actions available in the mini-cart. So we can actually put some content in the mini-cart with some functionality, and not edit the plugin file.

do_action( 'woocommerce_before_mini_cart' ); 
do_action( 'woocommerce_after_mini_cart' ); 

### Describe the solution you'd like

In your mini-cart file, please add the following lines: 

LINE 70:  do_action( 'woocommerce_before_mini_cart' );
LINE 102:  do_action( 'woocommerce_after_mini_cart' ); 

### Describe alternatives you've considered

Modifying elementor-pro

**Comment by stickypages** [comment]
Reference Image
![image](https://github.com/elementor/elementor/assets/658488/8c23b9a3-0379-43e6-b93b-88a058d2e333)

**Comment by louiswol94** [comment]
@stickypages Maybe you can use these hooks?

`woocommerce_before_template_part`
`woocommerce_after_template_part`

Example:
```
/**
 * Add some custom content after certain template parts.
 */
function my_custom_content_after_template_part( $name ) {
    if ( $name === 'cart/mini-cart.php' ) {
        echo '<p>Some text here.</p>';
    }
}
add_action( 'woocommerce_after_template_part', 'my_custom_content_after_template_part', 10, 1 );
```

**Reply by stickypages** [reply]
That's awesome, totally didn't think of the `woocommerce_after_template_part`. That works for now, I still think it might be goo to have a custom hook there. But that works for me, thanks!

**Reply by msjoker** [reply]
And to add it "here", is there also a hook available?

![Captura de pantalla 2024-08-19 a las 14 52 27](https://github.com/user-attachments/assets/f2f51d07-582a-45d2-a8fc-beff0bfa7f40)

Thanks!
